src/coreclr/src/scripts/genEventing.py

```diff
@@ -635,12 +635,11 @@ def generatePlatformIndependentFiles(sClrEtwAllMan, incDir, etmDummyFile, extern
         clrproviders = os.path.join(incDir, "clrproviders.h")
         with open_for_update(clrproviders) as Clrproviders:
             Clrproviders.write("""
-    typedef struct _EVENT_DESCRIPTOR
-    {
-        int const Level;
-        ULONGLONG const Keyword;
-    } EVENT_DESCRIPTOR;
-    """)
+typedef struct _EVENT_DESCRIPTOR
```

Comment:
A nit - was this change intentional?

Reply:
It was adding extra spaces and misaligned with rest of the file:

![image](https://user-images.githubusercontent.com/3840695/75187953-e5f28600-5753-11ea-948b-1125d7e651c4.png)

(I can revert this change if it is too irrelevant; since we have removed the checked-in dummy version of this file from previous commit: `src/coreclr/src/inc/clrprovidersdummy.h`)

Reply:
Actually, the thing that caught my eyes with your change is the  location of the final `"""`. But I don't feel strongly about it, so please feel free to leave it as-is.

Reply:
Ah right, `"""` can be at the beginning of next line to add a gap before next `#ifdef`. I will leave it as-is. Thank you.

Reply:
Moved `"""` on next line in #32800.